src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -357,7 +357,7 @@ private void Replicate()
                     if (_parent.DebugWaitAndRunReplicationOnce != null)
                     {
                         _parent.DebugWaitAndRunReplicationOnce.Wait(_cts.Token);
-                        _parent.DebugWaitAndRunReplicationOnce.Reset();
+                        _parent.DebugWaitAndRunReplicationOnce?.Reset();
```

Comment:
We checked that this is not null 2 lines up?

Reply:
in the test I do
```
database.ReplicationLoader.DebugWaitAndRunReplicationOnce = null;
mre.Set();
```